tests/components/__snapshots__/spreadsheet.test.ts.snap

```diff
@@ -525,44 +525,80 @@ height: 1px;
     class="o-spreadsheet-bottom-bar o-two-columns"
   >
     <div
-      class="o-sheet-item o-add-sheet"
+      class="o-ripple-container position-relative"
     >
-      <svg
-        class="o-icon"
+      <div
+        class="position-absolute"
+        style="
+top: NaNpx;
+left: NaNpx;
```

Comment:
Seems weird

Reply:
Yeah that's because I don't mock the `getBoudingClientRect()` here. Maybe I should mock them globally ? idk

Reply:
If it's easy, yes. If not, it's not a big issue